examples/chip-tool/commands/tests/TestCommand.cpp

```diff
@@ -20,46 +20,23 @@
 
 constexpr uint16_t kWaitDurationInSeconds = 30;
 
-static void test_os_sleep_ms(uint64_t millisecs)
```

Comment:
This is already going away in #7434, fwiw.  Would be better to land that separately so we have clearer regression tracking.